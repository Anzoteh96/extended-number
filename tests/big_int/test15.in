8072445910863100783563002301331147375290124222
-420403622518392965547340011548479718016781172
