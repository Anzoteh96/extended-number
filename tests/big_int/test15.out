Enter two numbers n and m:
n is 8072445910863100783563002301331147375290124222
m is -420403622518392965547340011548479718016781172
n + m is 7652042288344707818015662289782667657273343050
n - m is 8492849533381493749110342312879627093306905394
n * m is -3393685503510635890513745395035285264542032514956067733864580201331447756529621520470748184
n / m is -19
n % m is 84777083013634438163542081910032732971281954
